frontend/src/components/Credentials/GCredentialDetailsItemContent.vue

```diff
@@ -161,7 +165,7 @@ export default {
             return [
               {
                 label: 'Hetzner Cloud Token',
-                value: decodeBase64(secretData.hcloudToken),
+                value: 'hidden',
```

Comment:
what is the benefit of having the value "hidden" (which is not displayed with `font-weight-light text-disabled` class), vs. letting it run into the default case and return undefined.